custom-elements-manifest.config.js

```diff
@@ -4,7 +4,10 @@ import { pascalCase } from 'pascal-case';
 
 const packageData = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
 const { name, description, version, author, homepage, license } = packageData;
-const noDash = string => string.replace(/^\s?-/, '').trim();
```

Comment:
We can always add an eslint-disable-next-line comment in some cases. Should I go through and revert any one-liner functions back to arrow functions with disable comments?

Reply:
I apologize because this is largely coding style and opinion, but making this allowed — at least for short definitions like this one — will really help me out.